src/routes/safe/components/Transactions/TxList/QueueTxList.tsx

```diff
@@ -59,24 +59,33 @@ type QueueTransactionProps = {
   transactions: Transaction[]
 }
 
-const QueueTransaction = ({ nonce, transactions }: QueueTransactionProps): ReactElement =>
-  transactions.length > 1 ? (
-    <GroupedTransactionsCard>
+const QueueTransaction = ({ nonce, transactions }: QueueTransactionProps): ReactElement => {
+  const [nrChildrenExpanded, setHasChildrenExpanded] = useState(0)
+
+  const handleClickExpand = (expand: number) => {
```

Comment:
I would suggest renaming this to be a bit more declarative. It's not so clear what it does.